src/Uno.UI.Wasm/ts/Windows/Phone/Devices/Notification/VibrationDevice.ts

```diff
@@ -0,0 +1,8 @@
+﻿namespace Windows.Phone.Devices.Notification {
+
+	export class VibrationDevice {
+		public static vibrate(duration: number): boolean {
+			return window.navigator.vibrate(duration);
```

Comment:
Maybe a feature check should be done before calling it...
<https://davidwalsh.name/vibration-api>

Reply:
Nice find. I was searching for feature check and got to SO with an accepted answer that there is no way to check. Will implement this one.